modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -40,215 +66,246 @@ definition {
-			// // assert body element has "c-prefers-link-underline" class present
-			// //open accessibility menu
-			// // assert configuration is ON state
-			// Note: may need to remove any underline html attributes. see https://www.w3schools.com/howto/howto_js_remove_class.asp
 
-		}
+		//task ("Given When Then") {
+
+		// // Given create a new user
+		// // And as a guest user
+		// // // sign out of admin role to become a guest user role
+		// // And toggle show underline to ON state
+		// // //click show underline to ON state
+		// // When sign in as a new user for first time
+		// // Then configuration is persisted to first time signed in user
+		// // // assert body element has "c-prefers-link-underline" class present
+		// // //open accessibility menu
+		// // // assert configuration is ON state
+		// // Note: may need to remove any underline html attributes. see https://www.w3schools.com/howto/howto_js_remove_class.asp
+
+		//}
+
 	}
 
 	@description = "LPS-178192. Verifies accessibility menu quick link is keyboard focusable."
-	@ignore = "Test Stub"
-	@priority = 3
+	@priority = 5
 	test LinkIsKeyboardFocusable {
 		task ("When press tab key from 'Skip to Content' link") {
-
-			// Example:
-
-			//KeyPress(
-			//	key_text = "New Experience",
-			//	locator1 = "Button#ANY",
-			//	value1 = "\TAB");
-
+			AccessibilityMenu.goAccessibilityMenu();
 		}
 
 		task ("Then Accessibility Menu link is focusable") {
-
-			//TestUtils.assertElementFocused();
-
+			TestUtils.assertElementFocused(element = "//*[@aria-label='Quick Links']//li//*[contains(text(),'Open Accessibility Menu')]");
 		}
 	}
 
 	@description = "LPS-178192. Verifies accessibility menu modal can be accessed by keyboard."
 	@ignore = "Test Stub"
-	@priority = 3
+	@priority = 5
 	test ModalIsKeyboardAccessible {
-		task ("Given When Then") {
+		task ("When Accessibility Menu is focused with keyboard") {
+			AccessibilityMenu.goAccessibilityMenu();
+		}
 
-			// Given Accessibility Menu is focused with keyboard
-			// When access Accessibility Menu link with "Enter" key
-			// Then Accessibility Menu modal is present
-			// And can navigate to a switch toggle with keyboard
+		task ("And access Accessibility Menu link with 'Enter' key") {
+			KeyPress(
+				key_link = "Accessibility Menu",
+				locator1 = "AccessibilityMenu#QUICK_LINK",
+				value1 = "\ENTER");
+		}
 
+		task ("Then Accessibility Menu modal is present") {
+			AssertElementPresent(
+				key_modal_title = "Accessibility Help Menu",
+				locator1 = "AccessibilityMenu#MODAL_TITLE");
+		}
+
+		task ("And can navigate to a switch toggle with keyboard") {
+			KeyPress(
+				locator1 = "//body",
+				value1 = "\TAB");
+
+			KeyPress(
+				key_modalTitle = "Accessibility Help Menu",
+				locator1 = "Button#CLOSE_MODAL",
+				value1 = "\TAB");
+
+			TestUtils.assertElementFocused(element = "//label[contains(.,'Underlined Links')]//input[@type='checkbox' and not(@checked)]");
```

Comment:
ok for now, need to update this after new poshi function is implemented: https://liferay.slack.com/archives/CD7939WBE/p1686950932162979